### ISSUE TITLE
Commands for led display

### ISSUE THREAD

**Issue by thomassimon** [dillenberger/pepperl_fuchs#2]
Hi,
Can this driver be use to interact with the led display of the R2000 ? If not, do you have a clue to do that ?
Thanks.

**Comment by dillenberger** [comment]
No, you can't really interact with the LED display with this driver.
There is probably only one thing you can do (by changing a parameter): You can switch off the P+F logo, which is displayed after startup.

I'm in reguarly contact with the developers/distributors of the scanner. We also asked for a protocol for changing the logo displayed by the led display, but there is no such protocol implemented in the scanner as far as I know. Last time I talked to them was some months ago this year.
